content/operate/rc/databases/configuration/sizing.md

```diff
@@ -0,0 +1,81 @@
+---
+Title: Size a Redis Cloud Database
```

Comment:
```suggestion
Title: Size a Redis Cloud database
```
I don't think Database is part of the product name so in sentence case this wouldn't be capitalized